### DISCUSSION TITLE
开发一个监视剪贴板的功能

### DISCUSSION THREAD

**Discussion by ninja33** [ninja33/ODH#262] [Ideas]
我觉得可以试一下开发一个监视剪贴板的功能,即检测到剪贴板包含英文的话,自动查询剪贴板内容.Yomichan就实现了[类似功能](https://github.com/FooSoft/yomichan/commit/704864b7b2365de488150c947d50e27c97d3bc4c),可以作为参考.

_Originally posted by @proItheus in https://github.com/ninja33/ODH/issues/247#issuecomment-1229441140_

**Comment by proItheus** [comment]
请问您最近有开发这个功能的计划吗? @ninja33

**Reply by ninja33** [reply]
最近还没计划动手写，谁有兴趣可以fork了写一个，然后我这边可以merge

**Comment by extr15** [comment]
感谢老黄的工作 @ninja33  。很希望能有这个功能，可以在其他**任何软件**中复制出需要的句子，然后切换到chrome，浏览器中的剪贴板就能识别这个句子（可能还需要手动选下句子）这样就能方便的添加到anki。
这样扩展性很好，比如可以用其他阅读器看pdf，有需要时再查单词（我觉得复制一下句子这个操作不算麻烦）。

另，现在 浏览器中“文本输入框”中的单词 好像是没法查询的，不会有ODH弹窗，如果有这个功能，也就相当于实现了上面说的监视剪贴板
